hypervisor/debug/shell.c

```diff
@@ -79,52 +79,58 @@ static struct shell_cmd shell_cmds[] = {
 		.fcn		= shell_list_vcpu,
 	},
 	{
-		.str		= SHELL_CMD_VCPU_DUMPREG,
-		.cmd_param	= SHELL_CMD_VCPU_DUMPREG_PARAM,
-		.help_str	= SHELL_CMD_VCPU_DUMPREG_HELP,
-		.fcn		= shell_vcpu_dumpreg,
```

Comment:
Why you delete VCPU_DUMPREG?

Reply:
@dongyaozu  We didn't delete any command, just requeue the command, so that we use the help command, the command with no argument will be printed at first.

Reply:
This is a topic of which one should be printed earlier and what is the order.
How does commercial debugger do? I thought it can be in alpha-beta sequence or base on categories of meaning.  Sequence base on parameter # is rare case to me.

@gvancuts your comments?